ibtk/include/ibtk/LInitStrategy.h

```diff
@@ -111,6 +111,11 @@ class LInitStrategy : public virtual SAMRAI::tbox::DescribedClass
                                       bool can_be_refined,
                                       bool initial_time) = 0;
 
+    /*!
+     * \brief Initialize structure specific configurations.
+     */
+    virtual void init() = 0;
```

Comment:
Shall we provide a default empty implementation?

Reply:
With the default implementation, this method should not be purely virtual (i.e., please remove ` = 0`).